objects/src/testing/storage.rs

```diff
@@ -232,18 +185,18 @@ pub fn generate_account_seed(
             AccountType::NonFungibleFaucet,
         ),
         AccountSeedType::RegularAccountUpdatableCodeOnChain => (
-            mock_account(
-                ACCOUNT_ID_REGULAR_ACCOUNT_UPDATABLE_CODE_OFF_CHAIN,
-                Felt::ONE,
-                mock_account_code(assembler),
+            Account::mock(
+                ACCOUNT_ID_REGULAR_ACCOUNT_UPDATABLE_CODE_ON_CHAIN,
+                Felt::ZERO,
```

Comment:
was there a reason we changed from `Felt::ONE` to `Felt::ZERO`? Was there a reason it was one instead of zero before?